### PR TITLE
Legal archive UI

### PR DESCRIPTION
**Pull request by amagdas**

### CODE REVIEW

File: server/apps/legal_archive/resource.py [hunk 1]
```diff
@@ -18,41 +19,30 @@
...
 
-class LegalArchiveResource(Resource):
-    endpoint_name = LEGAL_ARCHIVE_NAME
+class LegalResource(Resource):
     schema = {}
     resource_methods = ['GET']
     item_methods = ['GET']
-    resource_title = endpoint_name
-    internal_resource = True
+    privileges = {'GET': LEGAL_ARCHIVE_NAME}
```
Comment (sivakuna-aap) [inline server/apps/legal_archive/resource.py]: This privilege will not work as by default `GET` is allowed for all users. For more information, please look https://github.com/superdesk/superdesk/blob/master/server/apps/auth/auth.py#L94

Comment (amagdas) [inline server/apps/legal_archive/resource.py]: Thanks, forgot about it, will fix.

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**